keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/NotificationEvent.kt

```diff
@@ -0,0 +1,13 @@
+package com.netflix.spinnaker.keel.events
+
+import com.netflix.spinnaker.keel.notifications.Notification
+import com.netflix.spinnaker.keel.notifications.NotificationScope
+import com.netflix.spinnaker.keel.notifications.NotificationType
+
+data class NotificationEvent(
+  val scope: NotificationScope,
+  val identifier: String,
```

Comment:
To avoid confusing this with an actual identifier for this object (which this is not), we've used the name `ref` in the `PersistentEvent` class, which has a similar model. I'd recommend following that pattern here as well.
```suggestion
  val ref: String, // the identifier of the persistent entity (resource, environment, etc.) this notification refers to
```